{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Multiple SPDX license identifiers found in source file. Use \"AND\" or \"OR\" to combine multiple licenses. Please see https://spdx.org for more information.\n--> /Users/giacomoibba/Desktop/SmarthER/mainnet/82/825B116b431c441C8f5eC19abC069adbd0a169cF_USDG.sol\n\nError: Source \"ERC20ElasticSupply.sol\" not found: File not found.\n --> /Users/giacomoibba/Desktop/SmarthER/mainnet/82/825B116b431c441C8f5eC19abC069adbd0a169cF_USDG.sol:4:1:\n  |\n4 | import \"ERC20ElasticSupply.sol\";\n  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"ERC20.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/82/825B116b431c441C8f5eC19abC069adbd0a169cF_USDG.sol:49:1:\n   |\n49 | import \"ERC20.sol\";\n   | ^^^^^^^^^^^^^^^^^^^\n\nError: Source \"Ownable.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/82/825B116b431c441C8f5eC19abC069adbd0a169cF_USDG.sol:50:1:\n   |\n50 | import \"Ownable.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"TimeLocks.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/82/825B116b431c441C8f5eC19abC069adbd0a169cF_USDG.sol:52:1:\n   |\n52 | import \"TimeLocks.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"IERC20.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/82/825B116b431c441C8f5eC19abC069adbd0a169cF_USDG.sol:221:1:\n    |\n221 | import \"IERC20.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"IERC20Metadata.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/82/825B116b431c441C8f5eC19abC069adbd0a169cF_USDG.sol:222:1:\n    |\n222 | import \"IERC20Metadata.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"Context.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/82/825B116b431c441C8f5eC19abC069adbd0a169cF_USDG.sol:223:1:\n    |\n223 | import \"Context.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"IERC20.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/82/825B116b431c441C8f5eC19abC069adbd0a169cF_USDG.sol:688:1:\n    |\n688 | import \"IERC20.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"Context.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/82/825B116b431c441C8f5eC19abC069adbd0a169cF_USDG.sol:742:1:\n    |\n742 | import \"Context.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"Ownable.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/82/825B116b431c441C8f5eC19abC069adbd0a169cF_USDG.sol:817:1:\n    |\n817 | import \"Ownable.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^\n\n\n", "results": {}}
